pyro/poutine/__init__.py

```diff
@@ -126,6 +143,25 @@ def condition(fn, data):
     return ConditionPoutine(fn, data=data)
 
 
+def continuation(fn, escape_fn, continuation_fn, first_available_dim=None):
```

Comment:
Do we really want to expose this? It seems weird exposing a detail like `first_available_dim` in the poutine interface, and I can't think of examples beyond `EnumeratePoutine`

Also nit: argument should be named `cont_fn`

Reply:
I was just following the convention of the rest of `poutine/__init__.py` in exposing all constructor args in the alias, will remove it and rename `continuation_fn`.